generate_stubs.py

```diff
@@ -29,6 +31,50 @@
 REFLECTED_OPERATOR_RE = regex.compile(r'- +`__r(.*)__`')
 ALIAS_RE = regex.compile(r'Alias for {ref}`function-(.*)`.')
 
+OPS = [
+    '__abs__',
+    '__add__',
+    '__and__',
+    '__bool__',
+    '__eq__',
+    '__float__',
+    '__floordiv__',
+    '__ge__',
+    '__getitem__',
+    '__gt__',
+    '__invert__',
+    '__le__',
+    '__lshift__',
+    '__lt__',
+    '__matmul__',
+    '__mod__',
+    '__mul__',
+    '__ne__',
+    '__neg__',
+    '__or__',
+    '__pos__',
+    '__pow__',
+    '__rshift__',
+    '__sub__',
+    '__truediv__',
+    '__xor__'
+]
```

Comment:
Maybe not super important since these are unlikely to change, but can't we extract this list from the spec?

Reply:
Seemingly not, as there needs to be a way to distinguish standard operators to in-place operators.